src/Playwright/Core/Page.cs

```diff
@@ -1153,6 +1153,15 @@ private FrameSetInputFilesOptions Map(PageSetInputFilesOptions options)
             };
         }
 
-        public Task RouteFromHARAsync(string har, PageRouteFromHAROptions options = null) => throw new NotImplementedException();
+        public async Task RouteFromHARAsync(string har, PageRouteFromHAROptions options = null)
+        {
+            var harRouter = await HarRouter.CreateAsync(Channel.Connection.LocalUtils, har, options?.NotFound, new()
```

Comment:
Default notFound to abort: https://github.com/microsoft/playwright/blob/ae6f48c4b8bb55db693dfdfd27592d9206e0d46d/packages/playwright-core/src/client/browserContext.ts#L274

Reply:
I do it inside the HARRouter, but let me move it up a level.